src/solver/batchnorm/backward_spatial_single.cpp

```diff
@@ -144,28 +144,18 @@ BnBwdTrainingSpatialSingle::GetSolution(const ExecutionContext& context,
         {
             if((n > 64) && (in_cstride > 160))
             {
-                variant = 3;
-                xlocalsize =
-                    std::min(64 * ((in_cstride + 63) / 64), static_cast<unsigned int>(1024));
-                xgridsize = c * xlocalsize;
-                ldsgcn    = xlocalsize / 64;
-                ldsnogcn  = xlocalsize;
+                variant    = 3;
+                xlocalsize = 1024;
+                xgridsize  = c * xlocalsize;
+                ldsgcn     = xlocalsize / 64;
```

Comment:
@muralinr Does 64 here mean WaveSize?

Reply:
It seems so from comments above: https://github.com/ROCmSoftwarePlatform/MIOpen/pull/1386#discussion_r790980027
CI has failed for some nan issues, we may need to deal with them first.

Reply:
> @muralinr Does 64 here mean WaveSize?

yes Jing.